### ISSUE TITLE
[WCAG] Style form control focus indicators to look like buttons focus indicators

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3630]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

In keyboard navigation mode (WCAG)...

The button focus indicators have already been updated to use a black outline. However, form controls use inconsistent styling.

<img width="617" height="501" alt="Image" src="https://github.com/user-attachments/assets/ef90ff63-662e-41bb-9078-fe20ae019959" />
<img width="539" height="238" alt="Image" src="https://github.com/user-attachments/assets/cd5fa08f-2e5b-4e5e-a9c0-1da4b096bb02" />

### Expected Behavior

All focus indicators shold have a similar style (black outline)

**Comment by kenchase** [comment]
MUI 9.4 has a theme opt-in for consistent keyboard focus rings. This will replace what's been done here.
